### PR TITLE
paramiko: set look_for_keys and allow_agent to False

### PR DESCRIPTION
**Pull request by lluiscampos**
Following changes from integration test framework. See:
* https://github.com/mendersoftware/integration/pull/1333
* https://github.com/mendersoftware/integration/pull/1336

### CODE REVIEW

File: tests/utils/fixtures/fixtures.py [hunk 1]
```diff
@@ -58,7 +58,13 @@ def config_host(host):
 
 def connection_factory(request, user, host, ssh_priv_key):
     host, port = config_host(host)
-    connect_kwargs = {"password": "", "banner_timeout": 60, "auth_timeout": 60}
+    connect_kwargs = {
+        "password": "",
+        "banner_timeout": 60,
+        "auth_timeout": 60,
+        "look_for_keys": False,
+        "allow_agent": False,
```
Comment (kacf) [inline tests/utils/fixtures/fixtures.py:66]: Does this prevent the password dialog during tests?

Comment (lluiscampos) [reply tests/utils/fixtures/fixtures.py:66]: Yes! :partying_face: 

I tested it locally and it works for me at least. We merged the same kind of fix on the integration test suite (see referred PRs above) and I haven't heard about tests not working locally for anybody.

I will do a full CI run before merge, though.

**Review by merlin-northern — APPROVED** [review]
lovely

**Comment by lluiscampos** [comment]
Tested in https://github.com/mendersoftware/meta-mender/pull/1355

File: tests/utils/fixtures/fixtures.py (hunk 1 above)
Comment (kacf) [reply tests/utils/fixtures/fixtures.py:66]: Nice, very much a wanted feature!

**Pull request merged**